source/bin/aws-instance-scheduler.ts

```diff
@@ -28,6 +28,9 @@ const SOLUTION_PROVIDER = 'AWS Solution Development';
 const app = new cdk.App();
 
 new AwsInstanceSchedulerStack(app, 'aws-instance-scheduler', {
+    synthesizer: new cdk.DefaultStackSynthesizer({
```

Comment:
Should this rule also be applied to the test stacks? Shouldn't affect tests, but no reason for them to be different.